src/main/java/tracko/logic/commands/order/FindOrderCommand.java

```diff
@@ -1,36 +1,43 @@
 package tracko.logic.commands.order;
 
+import static java.util.Objects.requireNonNull;
+
+import tracko.commons.core.Messages;
 import tracko.logic.commands.Command;
 import tracko.logic.commands.CommandResult;
 import tracko.model.Model;
-import tracko.model.order.NameContainsKeywordsPredicate;
+import tracko.model.order.OrderContainsKeywordsPredicate;
 
 /**
  * Finds and lists all persons in address book whose name contains any of the argument keywords.
  * Keyword matching is case insensitive.
  */
 public class FindOrderCommand extends Command {
 
-    public static final String COMMAND_WORD = "find";
+    public static final String COMMAND_WORD = "findo";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all orders that contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
             + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+            + "Example: " + COMMAND_WORD + " keychain";
 
-    private final NameContainsKeywordsPredicate predicate;
+    private final OrderContainsKeywordsPredicate predicate;
 
-    public FindOrderCommand(NameContainsKeywordsPredicate predicate) {
+    public FindOrderCommand(OrderContainsKeywordsPredicate predicate) {
         this.predicate = predicate;
     }
 
     @Override
     public CommandResult execute(Model model) {
-        // requireNonNull(model);
-        // model.updateFilteredPersonList(predicate);
-        // return new CommandResult(
-        //        String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, model.getFilteredPersonList().size()));
-        return null;
+        requireNonNull(model);
+        model.updateFilteredOrderList(predicate);
+
+        if (model.getFilteredOrderList().size() == 1) {
+            return new CommandResult(Messages.MESSAGE_ONE_ORDER_FOUND);
+        }
+
+        return new CommandResult(
+                String.format(Messages.MESSAGE_ORDERS_FOUND_OVERVIEW, model.getFilteredOrderList().size()));
```

Comment:
As mentioned above, can change the orders message to reduce number of lines of code here

Reply:
Changed!